signals.pde

```diff
@@ -69,89 +87,111 @@ interface TurnLogic {
   Iterable<Turn> getTail();
 }
 
+// Klasa za kretanje ravno
```

Comment:
Šta?????
Ovak na filozofskom objašnjavaju stvari, to nema ni značenje ni smisla, jel se autić ne kreće naprijed ako nema ove klase???
Ako ti ovaj kod nije jasan, samo izbriši komentare, možda spomeni `s_like.tmx` da ljudi znaju koji nivo odigrati da vide razliku između logika.